### PR TITLE
👷  update request method for verify tiles, remove warnings banner for app:exec

### PR DESCRIPTION
**Pull request by prklm10**
- Updating verify tiles from get to post.
- Removing warning for app:exec command.

### CODE REVIEW

File: packages/client/src/client.js [hunk 1]
```diff
@@ -450,7 +450,7 @@ export class PercyClient {
     this.log.debug(`Verifying comparison tile with sha: ${sha}`);
 
     try {
-      return await this.get(`comparisons/${comparisonId}/tiles/verify`, {
+      return await this.post(`comparisons/${comparisonId}/tiles/verify`, {
```
Comment (ninadbstack) [inline packages/client/src/client.js:453]: Shouldn't we have a test change related to this ?

File: packages/cli-app/src/app.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import exec from './exec.js';
 
 export const app = command('app', {
   description: 'Create Percy builds for native app snapshots',
-  hidden: 'This command is still in development and may not work as expected',
+  hidden: true,
```
Comment (itsjwala) [inline packages/cli-app/src/app.js:6]: * was there any test for this? 
* Have you tested it locally? - follow README on how to test it if not already( `yarn global:link`)

File: packages/client/src/client.js (hunk 1 above)
Comment (prklm10) [reply packages/client/src/client.js:453]: there is no specific test for the the request made in client.js. There are test written for this function. and there are separate tests written for get and post request.

File: packages/cli-app/src/app.js (hunk 2 above)
Comment (prklm10) [reply packages/cli-app/src/app.js:6]: - There was no test for this.
- There is test written for hidden in cli-command [test](https://github.com/percy/cli/blob/master/packages/cli-command/test/help.test.js#L280).
- I have tested locally and ran a build locally as well.
- I did `yarn build` and from `node_module/.bin/percy` i ran command to test it.

Comment (itsjwala) [reply packages/cli-app/src/app.js:6]: * can you take a stab and add a test in `cli-command`?

Comment (prklm10) [reply packages/cli-app/src/app.js:6]: added test.

File: packages/client/src/client.js (hunk 1 above)
Comment (prklm10) [reply packages/client/src/client.js:453]: added test

**Review by ninadbstack — APPROVED** [review]

**Review by itsjwala — APPROVED** [review]

**Pull request merged**